File: TI1/TI1.3 RESTFUL_JAVA_SINBDD_GR09/02.CLIESC/CONUNI_RESTFUL_JAVA_CLIESC_G09/src/main/java/ec/edu/monster/modelo/ConversionRequest.java
package ec.edu.monster.modelo;

public class ConversionRequest {
    private double valor;
    private String unidadOrigen;
    private String unidadDestino;
    private String categoria;

    public ConversionRequest() {}

    public ConversionRequest(double valor, String unidadOrigen, String unidadDestino, String categoria) {
        this.valor = valor;
        this.unidadOrigen = unidadOrigen;
        this.unidadDestino = unidadDestino;
        this.categoria = categoria;
    }

    public double getValor() {
        return valor;
    }

    public void setValor(double valor) {
        this.valor = valor;
    }

    public String getUnidadOrigen() {
        return unidadOrigen;
    }

    public void setUnidadOrigen(String unidadOrigen) {
        this.unidadOrigen = unidadOrigen;
    }

    public String getUnidadDestino() {
        return unidadDestino;
    }

    public void setUnidadDestino(String unidadDestino) {
        this.unidadDestino = unidadDestino;
    }

    public String getCategoria() {
        return categoria;
    }

    public void setCategoria(String categoria) {
        this.categoria = categoria;
    }
}

File: TI1/TI1.2 SOAP_DOTNET_SINBDD_GR09/02.CLIWEB/CONUNI_SOAP_DOTNET_CLIWEB_G09/CONUNI_SOAP_DOTNET_CLIWEB_G09/Models/ConversionRequest.cs
namespace CONUNI_SOAP_DOTNET_CLIWEB_G09.Models
{
    /// <summary>
    /// Modelo para las peticiones de conversión
    /// </summary>
    public class ConversionRequest
    {
        public double Valor { get; set; }
        public string UnidadOrigen { get; set; } = string.Empty;
        public string UnidadDestino { get; set; } = string.Empty;
        public string Categoria { get; set; } = string.Empty;
    }
}


